zerver/tests/test_create_video_call.py

```diff
@@ -129,18 +139,18 @@ def test_create_video_request_error(self) -> None:
         self.assertEqual(response.status_code, 200)
 
         response = self.client_post("/json/calls/zoom/create")
-        self.assert_json_error(response, "Failed to create Zoom call")
+        self.assert_json_error(response, "Failed to create call")
 
         responses.replace(
-            responses.POST, "https://api.zoom.us/v2/users/me/meetings", status=401,
+            responses.POST, "https://zoom.example.com/v2/users/me/meetings", status=401,
         )
 
         response = self.client_post("/json/calls/zoom/create")
-        self.assert_json_error(response, "Invalid Zoom access token")
+        self.assert_json_error(response, "Invalid access token")
 
     @responses.activate
     def test_deauthorize_zoom_user(self) -> None:
-        responses.add(responses.POST, "https://api.zoom.us/oauth/data/compliance")
+        responses.add(responses.POST, "https://zoom.example.com/oauth/data/compliance")
```

Comment:
Generally, we aim to do this sort of refactor (along with the settings and structural changes to the Zoom integration) as a series of independent, atomic, preparatory commits, so that we can easily review and merge those refactors that are relatively fast to review while we do any back-and-forth on the implementation for the new Webex feature.  This ends up saving a lot of reviewer time for any PR that needs multiple review round trips.

Check out the Zulip commit guidelines for more details: https://zulip.readthedocs.io/en/latest/contributing/version-control.html

Reply:
Hi Tim, I've broken this out into 7 commits now: Refactor these tests, refactor the zoom_xhrs method, refactor the zoom API into a generic base, and then 4 stages of adding the new Webex stuff (token, api, frontend, documentation).